### PR TITLE
Metadata validation

### PR DESCRIPTION
**Pull request by tethryus**
Align content for metadata validation

### CODE REVIEW

**Comment by tethryus** [comment]
@bonczidai please review

File: content/io/cloudslang/base/examples/parallel_loop/create_directory.sl [hunk 1]
```diff
@@ -4,6 +4,8 @@
 #!
 #! @input directory_name: name of directory to be created
 #! @output error_msg: error message
+#! @result SUCCESS: directory created successfully
+#! @result FAILURE: something went wrong
```
Comment (gituser173) [inline content/io/cloudslang/base/examples/parallel_loop/create_directory.sl]: "there was an error creating the directory" WDYT? same comment for create_directories.sl

**Comment by Bonczidai** [comment]
@tethryus 💯 

I think you should merge from latest master to be sure that all content is updated, enable the `-des` flag on this branch (both circle and travis)

then we can merge the PR 💪

File: content/io/cloudslang/cloud/heroku/applications/delete_application.sl [hunk 2]
```diff
@@ -9,6 +9,8 @@
 #! @output error_message: return_result if status_code is not '200'
 #! @output return_code: '0' if success, '-1' otherwise
 #! @output status_code: code returned by the operation
+#! @result SUCCESS: Heroku application deleted successfully
+#! @result FAILURE: there was an error wihle trying to delete Heroku application
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/heroku/applications/delete_application.sl]: while

File: content/io/cloudslang/cloud/heroku/collaborators/delete_application_collaborator.sl [hunk 3]
```diff
@@ -10,6 +10,8 @@
 #! @output error_message: return_result if status_code is not '200'
 #! @output return_code: '0' if success, '-1' otherwise
 #! @output status_code: code returned by operation
+#! @result SUCCESS: Heroku application collaborator deleted successfully
+#! @result FAILURE: there was an error wihle trying to delete Heroku application collaborator
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/heroku/collaborators/delete_application_collaborator.sl]: while

File: content/io/cloudslang/cloud/heroku/collaborators/list_application_collaborators.sl [hunk 4]
```diff
@@ -9,6 +9,8 @@
 #! @output error_message: return_result if status_code is not '200'
 #! @output return_code: '0' if success, '-1' otherwise
 #! @output status_code: code returned by the operation
+#! @result SUCCESS: Heroku collaborators for a specigic application retrieved successfully
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/heroku/collaborators/list_application_collaborators.sl]: specific

File: content/io/cloudslang/cloud/heroku/configvars/get_application_config_vars.sl [hunk 5]
```diff
@@ -9,6 +9,8 @@
 #! @output error_message: return_result if status_code is not '200'
 #! @output return_code: '0' if success, '-1' otherwise
 #! @output status_code: code returned by the operation
+#! @result SUCCESS: Heroku application varaiables retrieved successfully
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/heroku/configvars/get_application_config_vars.sl]: variables

File: content/io/cloudslang/cloud/heroku/keys/list_keys.sl [hunk 6]
```diff
@@ -8,6 +8,8 @@
 #! @output error_message: return_result if status_code is not '200'
 #! @output return_code: '0' if success, '-1' otherwise
 #! @output status_code: code returned by the operation
+#! @result SUCCESS: existing users's SSH keys retrieved successfully
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/heroku/keys/list_keys.sl]: Either user's for singular or users' for plural. (Same in next line.)

File: content/io/cloudslang/cloud/openshift/applications/delete_application.sl [hunk 7]
```diff
@@ -29,6 +29,8 @@
 #! @output error_message: return_result if statusCode is not '200'
 #! @output return_code: '0' if success, '-1' otherwise
 #! @output status_code: code returned by the operation
+#! @result SUCCESS: Openshit application deleted successfully
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openshift/applications/delete_application.sl]: OpenShift

File: content/io/cloudslang/cloud/openshift/applications/list_applications.sl [hunk 8]
```diff
@@ -28,6 +28,8 @@
 #! @output error_message: return_result if status_code is not '200'
 #! @output return_code: '0' if success, '-1' otherwise
 #! @output status_code: code returned by the operation
+#! @result SUCCESS: Openshift applications from the specific domain retrieved successfully
+#! @result FAILURE: thre was an error while trying to retrieve Openshift applications from a specified domain
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openshift/applications/list_applications.sl]: there

File: content/io/cloudslang/cloud/openshift/applications/restart_application.sl [hunk 9]
```diff
@@ -29,6 +29,8 @@
 #! @output error_message: return_result if status_code is not '200'
 #! @output return_code: '0' if success, '-1' otherwise
 #! @output status_code: code returned by the operation
+#! @result SUCCESS: Openshit application restarted successfully
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openshift/applications/restart_application.sl]: OpenShift

File: content/io/cloudslang/cloud/openshift/cartridges/add_cartridge.sl [hunk 10]
```diff
@@ -33,6 +33,8 @@
 #! @output error_message: return_result if statusCode is not '201'
 #! @output return_code: '0' if success, '-1' otherwise
 #! @output status_code: code returned by the operation
+#! @result SUCCESS: embedded cartridge successfully added to the Openshift application
+#! @result FAILURE: there was an error wihle trying to add the embedded cartridge to the Openshift application
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openshift/cartridges/add_cartridge.sl]: while

File: content/io/cloudslang/cloud/openshift/cartridges/restart_cartridge.sl [hunk 11]
```diff
@@ -30,6 +30,8 @@
 #! @output error_message: return_result if status_code is not '200'
 #! @output return_code: '0' if success, '-1' otherwise
 #! @output status_code: code returned by the operation
+#! @result SUCCESS: embedded cartridge restarted successfully
+#! @result FAILURE: there was an error while trying to restart the running embeddded cartridge
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openshift/cartridges/restart_cartridge.sl]: embedded

File: content/io/cloudslang/cloud/openstack/openstack_health_check.sl [hunk 12]
```diff
@@ -26,16 +26,33 @@
...
-#! @result GET_TENANT_ID_FAILURE:
-#! @result GET_AUTHENTICATION_FAILURE:
-#! @result GET_SERVERS_FAILURE:
-#! @result EXTRACT_SERVERS_FAILURE:
-#! @result CHECK_SERVER_FAILURE:
-#! @result SEND_EMAIL_FAILURE:
-#! @result FAILURE:
+#! @result SUCCESS: server created, checked and deleted successfully
+#! @result CREATE_SERVER_FAILURE: there was an error while trying to create the server
+#! @result GET_AUTHENTICATION_TOKEN_FAILURE: there was an error wihle trying to get authentification token
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/openstack_health_check.sl]: while...authentication

File: content/io/cloudslang/cloud/openstack/openstack_health_check.sl [hunk 13]
```diff
@@ -26,16 +26,33 @@
...
+#! @result GET_TENANT_ID_FAILURE: there was an error wihle trying to get tenant ID
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/openstack_health_check.sl]: while

File: content/io/cloudslang/cloud/openstack/openstack_health_check.sl [hunk 14]
```diff
@@ -26,16 +26,33 @@
...
+#! @result GET_AUTHENTICATION_FAILURE: there was an error while trying to authentigicate
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/openstack_health_check.sl]: authenticate

File: content/io/cloudslang/cloud/openstack/openstack_health_check.sl [hunk 15]
```diff
@@ -26,16 +26,33 @@
...
+#! @result GET_SERVERS_FAILURE: there was an error while trying to retrieve servers
+#! @result EXTRACT_SERVERS_FAILURE: there was an error wihle trying to extract servers
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/openstack_health_check.sl]: while

File: content/io/cloudslang/cloud/openstack/openstack_health_check.sl [hunk 16]
```diff
@@ -26,16 +26,33 @@
...
+#! @result CHECK_SERVER_FAILURE: there was an error while trying to check the server
+#! @result SEND_EMAIL_FAILURE: there was an error while trying to send email
+#! @result FAILURE: something went wrong
+#! @result ADD_NAME_FAILURE: there was an error while trying to add server name
+#! @result ADD_IMG_REF_FAILURE: there was an error while trying to add image reference
+#! @result ADD_FLAVOR_REF_FAILURE: there was an error while trying to add flavor reference
+#! @result ADD_AVAILABILITY_ZONE_FAILURE: there was an error while trying to add availability zone
+#! @result ADD_KEY_NAME_FAILURE: there was an error while trying to add key anem
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/openstack_health_check.sl]: name

File: content/io/cloudslang/cloud/openstack/openstack_health_check.sl [hunk 17]
```diff
@@ -26,16 +26,33 @@
...
+#! @result ADD_ADMIN_PASS_FAILURE: there was an error wihle trying to set up admin password
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/openstack_health_check.sl]: while

File: content/io/cloudslang/cloud/openstack/openstack_health_check.sl [hunk 18]
```diff
@@ -26,16 +26,33 @@
...
+#! @result ADD_USER_DATA_FAILURE: there was an error while trying to set user data
+#! @result ADD_NETWORK_FAILURE: there was an error while trying to add server to the network
+#! @result ADD_METADATA_FAILURE: there was an error while trying to add metadata
+#! @result ADD_DEFAULT_SECURITY_GROUP_FAILURE: there was an error while trying to set up default security group
+#! @result ADD_SECURITY_GROUPS_FAILURE: there was an error while trying to add security groups
+#! @result ADD_BOOT_INDEX_FAILURE: there was an error while trying to set boot index
+#! @result ADD_UUID_FAILURE: there was an error while trying to add UUID
+#! @result ADD_SOURCE_TYPE_FAILURE: there was an error wihle trying to add srouce type
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/openstack_health_check.sl]: while...source

File: content/io/cloudslang/cloud/openstack/openstack_health_check.sl [hunk 19]
```diff
@@ -26,16 +26,33 @@
...
+#! @result ADD_DELETE_ON_TERMINATION_FAILURE: there was an error while trying to delete on termination
+#! @result ADD_BLOCK_DEVICE_MAPPING_FAILURE: there was an error wihle trying to add block device mapping
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/openstack_health_check.sl]: while

File: content/io/cloudslang/cloud/openstack/openstack_health_check.sl [hunk 20]
```diff
@@ -26,16 +26,33 @@
...
+#! @result ADD_PERSONALITY_FAILURE: there was an error wihle trying to set up personality
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/openstack_health_check.sl]: while

File: content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl [hunk 21]
```diff
@@ -73,25 +73,35 @@
...
-#! @result ADD_USER_DATA_FAILURE
-#! @result ADD_AVAILABILITY_ZONE_FAILURE
-#! @result ADD_NETWORK_FAILURE
-#! @result ADD_METADATA_FAILURE
-#! @result ADD_PERSONALITY_FAILURE
-#! @result ADD_INSTANCE_SOURCE_BOOT_FAILURE
-#! @result ADD_SERVER_OBJECT_JSON_FAILURE
-#! @result CREATE_SERVER_FAILURE
+#! @result SUCCESS: server created successfully
+#! @result GET_AUTHENTICATION_FAILURE: there was an error while retrieving authentification
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl]: authentication

File: content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl [hunk 22]
```diff
@@ -73,25 +73,35 @@
...
+#! @result GET_AUTHENTICATION_TOKEN_FAILURE: there was an error while retrieving token authentification
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl]: authentication

File: content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl [hunk 23]
```diff
@@ -73,25 +73,35 @@
...
+#! @result GET_TENANT_ID_FAILURE: there was an error while trying to get the tenant ID
+#! @result ADD_NAME_FAILURE: there was an error while trying to add server name
+#! @result ADD_IMG_REF_FAILURE: there was an error while trying to add image reference to the server
+#! @result ADD_FLAVOR_REF_FAILURE: there was an error wihle trying to add flavor reference to the server
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl]: while

File: content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl [hunk 24]
```diff
@@ -73,25 +73,35 @@
...
+#! @result ADD_KEY_NAME_FAILURE: there was an error wihle trying to add key name to the server
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl]: while

File: content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl [hunk 25]
```diff
@@ -73,25 +73,35 @@
...
+#! @result ADD_SECURITY_GROUPS_FAILURE: there was an error while trying to add the security group to the server
+#! @result ADD_ADMIN_PASS_FAILURE: there was an error while trying to add the admin password
+#! @result ADD_USER_DATA_FAILURE: there was an error while trying to add user data
+#! @result ADD_AVAILABILITY_ZONE_FAILURE: there was an error while trying to add availability zone
+#! @result ADD_DEFAULT_SECURITY_GROUP_FAILURE: there was an error while trying to add the default security gorup
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl]: group

File: content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl [hunk 26]
```diff
@@ -73,25 +73,35 @@
...
+#! @result ADD_SECURITY_GROUPS_FAILURE: there was an error while trying to add the security groups
+#! @result ADD_BOOT_INDEX_FAILURE: there was an error while trying to add boot index
+#! @result ADD_DELETE_ON_TERMINATION_FAILURE: there was an error while trying to add delete on termination
+#! @result ADD_NETWORK_FAILURE: there was an error while trying to add network
+#! @result ADD_METADATA_FAILURE: there was an error while trying to add metadata
+#! @result ADD_PERSONALITY_FAILURE: there was an error while trying to add personality
+#! @result ADD_UUID_FAILURE: there was an error while trying to add the UUID
+#! @result ADD_SOURCE_TYPE_FAILURE: there was an error while trying to add source type
+#! @result ADD_BLOCK_DEVICE_MAPPING_FAILURE: there was an error wihle tryint to add the device mapping
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl]: while trying

File: content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl [hunk 27]
```diff
@@ -73,25 +73,35 @@
...
+#! @result ADD_INSTANCE_SOURCE_BOOT_FAILURE: there was an error while trying to add instance source boot
+#! @result ADD_SERVER_OBJECT_JSON_FAILURE: there was an error while trying to add server JSON object
+#! @result CREATE_SERVER_FAILURE: there was an error wihle trying to create server
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/servers/create_server_flow.sl]: while

File: content/io/cloudslang/cloud/openstack/utils/create_boot_json.sl [hunk 28]
```diff
@@ -15,12 +15,15 @@
...
-#! @output json_output: the boot specific JSON object
+#! @output boot_json: the boot specific JSON object
 #! @output return_result: contains the exception in case of failure, success message otherwise
 #! @output return_code: "0" if success, "-1" otherwise
 #! @output error_message: error message if there was an error when executing, empty otherwise
 #! @result SUCCESS: successfully created a boot specific JSON object (return_code == '0')
-#! @result FAILURE: otherwise
+#! @result ADD_BOOT_INDEX_FAILURE: there was an error while trying to add the boot index
+#! @result ADD_UUID_FAILURE: there was an error while trying to add the UUID
+#! @result ADD_SOURCE_TYPE_FAILURE: there was an error wihle trying to add the source type
```
Comment (SamMarkowitz) [inline content/io/cloudslang/cloud/openstack/utils/create_boot_json.sl]: while

File: content/io/cloudslang/coreos/list_machines_public_ip.sl [hunk 29]
```diff
@@ -16,6 +16,8 @@
 #! @input private_key_file: optional - path to the private key file
 #! @input timeout: optional - time in milliseconds to wait for the command to complete
 #! @output machines_public_ip_list: list of public IP addresses of the machines in the cluster (delimiter: space)
+#! @result SUCCESS: public IPs retrieved successfully
+#! @result FAILURE: there was an error wihle trying to retrieve the public IPs from the servers
```
Comment (SamMarkowitz) [inline content/io/cloudslang/coreos/list_machines_public_ip.sl]: while

File: content/io/cloudslang/docker/containers/examples/demo_dev_ops.sl [hunk 30]
```diff
@@ -28,6 +28,8 @@
 #! @input timeout: optional - time in milliseconds to wait for command to complete - Default: 30000000 ms (8.33 h)
 #! @input proxy_host: optional - proxy server used to access the web site
 #! @input proxy_port: optional - proxy server port
+#! @result SUCCESS: Docker containers linked successfully
+#! @result FAILURE: there was an error while tyring to link Docker containers
```
Comment (SamMarkowitz) [inline content/io/cloudslang/docker/containers/examples/demo_dev_ops.sl]: trying

File: content/io/cloudslang/docker/containers/get_running_processes.sl [hunk 31]
```diff
@@ -26,6 +26,8 @@
 #! @input agent_forwarding: optional - the sessionObject that holds the connection if the close session is false - Default: ''
 #! @output standard_err: error message
 #! @output return_result: list of running processes on the container
+#! @result SUCCESS: the list of running processes on that specific container retrieved sucessfully
+#! @result FAILURE: there was an error wihle trying to retrieve the list of running processes from the container
```
Comment (SamMarkowitz) [inline content/io/cloudslang/docker/containers/get_running_processes.sl]: while

File: content/io/cloudslang/docker/containers/start_container.sl [hunk 32]
```diff
@@ -24,6 +24,8 @@
 #!                       if 'true' the SSH session used will be closed; Valid: true, false
 #! @output container_id_output: ID of the container that was started
 #! @output error_message: error message
+#! @result SUCCESS: Docker container sucessfully started
+#! @result FAILURE: there was an error wihle trying to start the Docker container
```
Comment (SamMarkowitz) [inline content/io/cloudslang/docker/containers/start_container.sl]: while

File: content/io/cloudslang/docker/images/get_image_parent.sl [hunk 33]
```diff
@@ -16,6 +16,8 @@
 #! @input timeout: optional - time in milliseconds to wait for the command to complete
 #! @input port: optional - port number for running the command
 #! @output parent_image_name: name of the parent image
+#! @result SUCCESS: image`s parent inspected successfully
```
Comment (SamMarkowitz) [inline content/io/cloudslang/docker/images/get_image_parent.sl]: image's

File: content/io/cloudslang/docker/runc/examples/transfer_images.sl [hunk 34]
```diff
@@ -11,14 +11,17 @@
...
+#! @input target_container: destination container where the image will be transferred
 #! @input root_path: the full path to the folder which contains the containers folders . - Example: "/usr/local/migrate/"
 #! @input predump_image_location: the full path to the folder which will contain the container's pre_dump image.
 #! @input dump_image_location: the full path  to the folder which will contain the container's dump image.
-#! @result SUCCESS:
-#! @result PACK_DUMP_FAILURE:
-#! @result TRANSFER_DUMP_FAILURE:
-#! @result DELETE_DUMP_FAILURE:
+#! @result SUCCESS: image archived and transferred successfully
+#! @result PACK_DUMP_FAILURE: DUMNP packing failure
```
Comment (SamMarkowitz) [inline content/io/cloudslang/docker/runc/examples/transfer_images.sl]: DUMP

File: content/io/cloudslang/git/git_merge.sl [hunk 35]
```diff
@@ -27,6 +27,8 @@
 #!                              Examples: '0' for a successful command, '-1' if the command was not yet terminated (or this
 #!                              channel type has no command), '126' if the command cannot execute
 #! @output return_code: return code of the command
+#! @result SUCCESS: GIT branch mergedd successfully to another branch
```
Comment (SamMarkowitz) [inline content/io/cloudslang/git/git_merge.sl]: merged

File: content/io/cloudslang/haven_on_demand/format_conversion/text_extraction.sl [hunk 36]
```diff
@@ -9,6 +9,7 @@
 #!!
 #! @description: Extracts metadata and text content from a file.
 #! @input api_key: API Key
+#! @input text_extraction_api: URL to the HoN APi
```
Comment (SamMarkowitz) [inline content/io/cloudslang/haven_on_demand/format_conversion/text_extraction.sl]: I think this one should be `private: true` in the code and removed from the documentation. If not, it should say `HoD API`.

File: content/io/cloudslang/haven_on_demand/search/find_related_concepts.sl [hunk 37]
```diff
@@ -40,6 +40,8 @@
 #!                    optional
 #! @output return_result: result of API
 #! @output error_message: error message if one exists, empty otherwise
+#! @result SUCCESS: best phrases and terms retrieved successfully
+#! @result FAILURE: there was an error while tyring to retrieve best phrases and terms
```
Comment (SamMarkowitz) [inline content/io/cloudslang/haven_on_demand/search/find_related_concepts.sl]: trying

File: content/io/cloudslang/haven_on_demand/search/query_text_index.sl [hunk 38]
```diff
@@ -88,6 +88,8 @@
 #!                    optional
 #! @output return_result: result of API
 #! @output error_message: error message if one exists, empty otherwise
+#! @result SUCCESS: HoN database content retrieved successfully
```
Comment (SamMarkowitz) [inline content/io/cloudslang/haven_on_demand/search/query_text_index.sl]: HoD

File: content/io/cloudslang/haven_on_demand/search/query_text_index.sl [hunk 39]
```diff
@@ -88,6 +88,8 @@
...
+#! @result FAILURE: there was an error while trying to retrieve HoN database content
```
Comment (SamMarkowitz) [inline content/io/cloudslang/haven_on_demand/search/query_text_index.sl]: HoD

File: content/io/cloudslang/marathon/create_app.sl [hunk 40]
```diff
@@ -17,6 +17,8 @@
 #! @output status_code: normal status code is 200
 #! @output return_code: if return_code == -1 then there was an error
 #! @output error_message: return_result if return_code == -1 or status_code != 200
+#! @result SUCCESS: new Marathon app created with success
```
Comment (SamMarkowitz) [inline content/io/cloudslang/marathon/create_app.sl]: successfully

File: content/io/cloudslang/operations_orchestration/create_cp/create_package.sl [hunk 41]
```diff
@@ -13,19 +13,18 @@
...
-#! @result CREATE_LIBRARY_STRUCTURE_FAILURE:
-#! @result COPY_CONTENT_FAILURE:
-#! @result MOVE_CONFIG_ITEMS_FAILURE:
-#! @result CREATE_CP_PROPERTIES_FAILURE:
-#! @result CREATE_ARCHIVE_FAILURE:
-#! @result CREATE_JAR_FAILURE:
-#! @result CLEAN_FOLDER_FAILURE:
+#! @result SUCCESS: Content pack created successfully
+#! @result CREATE_LIB_FOLDER_FAILURE: there was an error while creting the LIB folder
+#! @result POPULATE_LIB_FOLDER_FAILURE: there was an error wihle trying to populate the LIB folder
```
Comment (SamMarkowitz) [inline content/io/cloudslang/operations_orchestration/create_cp/create_package.sl]: while

File: content/io/cloudslang/operations_orchestration/create_cp/create_package.sl [hunk 42]
```diff
@@ -13,19 +13,18 @@
...
+#! @result CREATE_SYSTEM_PROPERTIES_FAILURE: there was an error wihle trying to create the system properties file
```
Comment (SamMarkowitz) [inline content/io/cloudslang/operations_orchestration/create_cp/create_package.sl]: while

File: content/io/cloudslang/operations_orchestration/create_cp/create_package.sl [hunk 43]
```diff
@@ -13,19 +13,18 @@
...
+#! @result CREATE_LIBRARY_STRUCTURE_FAILURE: there was an error while trying to create the library structure
+#! @result COPY_CONTENT_FAILURE: there was an error wihle trying to copy content
```
Comment (SamMarkowitz) [inline content/io/cloudslang/operations_orchestration/create_cp/create_package.sl]: while

File: content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl [hunk 44]
```diff
@@ -90,9 +90,34 @@
 #! @input email_recipient: email recipient
 #! @output return_result: contains the exception in case of failure, success message otherwise
 #! @output return_code: '0' if operation was successfully executed, '-1' otherwise
-#! @output error_message: error message if there was an error when executing, empty otherwise
+#! @output exception: exception if there was an error when executing, empty otherwise
+#! @output before_update_value:
```
Comment (SamMarkowitz) [inline content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl]: no description (and on the next line as well)

File: content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl [hunk 45]
```diff
@@ -90,9 +90,34 @@
...
+#! @output after_update_value:
 #! @result SUCCESS: virtual machine was successfully created
-#! @result FAILURE: an error occurred when trying to create a new virtual machine
+#! @result GET_OS_DESCRIPTORS_FAILURE: there was an error while trying to get OS information
+#! @result SEND_OSES_SUPPORTED_LIST_MAIL_FAILURE: there was an error while sending an email with the OSes list
+#! @result GUEST_OS_ID_NOT_FOUND: there was an error while retrieving the gues OS UD
```
Comment (SamMarkowitz) [inline content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl]: guest OS ID

File: content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl [hunk 46]
```diff
@@ -90,9 +90,34 @@
...
+#! @result CREATE_VIRTUAL_MACHINE_FAILURE: there was an error wihle trying to create the virtual machine
```
Comment (SamMarkowitz) [inline content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl]: while

File: content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl [hunk 47]
```diff
@@ -90,9 +90,34 @@
...
+#! @result GET_CREATED_TEXT_OCCURRENCE_FAILURE: there was an error while retrieving the created text occurence
+#! @result LIST_VMS_FAILURE: there was an error while listing the virtual machines
+#! @result VM_NOT_FOUND: there was an error finding a virtual machine
+#! @result GET_CREATED_VM_DETAILS_FAILURE: there was an error while retrieving the
```
Comment (SamMarkowitz) [inline content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl]: end of description missing

File: content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl [hunk 48]
```diff
@@ -90,9 +90,34 @@
...
+#! @result SEND_CREATED_VM_MAIL_FAILURE: there was an error while trying to send an email with the VM details
+#! @result GET_VALUE_BEFORE_UPDATE_FAILURE: there was an error while retrieving the value before update
+#! @result UPDATE_VM_FAILURE: there was an error while updating the virtual machine
+#! @result GET_UPDATED_VM_DETAILS_FAILURE: there was an error while retrieving the updated VM details
+#! @result GET_VALUE_AFTER_UPDATE_FAILURE: there was an error while retrieving the value after the update
+#! @result SEND_UPDATED_VM_MAIL_FAILURE: ther ewas an error while sending an email with the updated VM details
```
Comment (SamMarkowitz) [inline content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl]: there was

File: content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl [hunk 49]
```diff
@@ -90,9 +90,34 @@
...
+#! @result POWER_ON_VM_FAILURE: there was an error while powering on the virtual machine
+#! @result NOT_POWERED_ON: the virtual machine is not powered on
+#! @result SEND_POWERED_ON_VM_MAIL_FAILURE: there was an error while trying to send an email about th VM power on state
```
Comment (SamMarkowitz) [inline content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl]: the

File: content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl [hunk 50]
```diff
@@ -90,9 +90,34 @@
...
+#! @result POWER_OFF_VM_FAILURE: there was an error while powering off the virtual machine
+#! @result NOT_POWERED_OFF: the virtual machine is not powered off
+#! @result SEND_POWERED_OFF_VM_MAIL_FAILURE: there was an error while trying to send an email about the VM power off state
+#! @result DELETE_VM_FAILURE: there was an error wihle deleting the virtual machine
```
Comment (SamMarkowitz) [inline content/io/cloudslang/virtualization/vmware/samples/virtual_machine_lifecycle.sl]: while

**Pull request merged**